src/extension/app/plugins/delete/delete.js

```diff
@@ -48,15 +48,6 @@ export function createDeletePlugin(appStore) {
         const isPage = status.webPath.split('/').pop().indexOf('.') === -1;
         const hasSrc = status.edit?.status === 200;
 
-        // double check
-        if (hasSrc && !appStore.isAuthenticated()) {
```

Comment:
Should never be able to call `delete` if authenticated

Reply:
@rofe But you would never see the delete plugin if you are not authenticated no?

Reply:
As discussed, leaving as is